src/v2/Apps/Components/AppShell.tsx

```diff
@@ -62,17 +63,21 @@ export const AppShell: React.FC<AppShellProps> = props => {
         </Box>
       </Box>
 
-      <Box as="main" id="main">
-        {children}
-      </Box>
+      <Theme theme={routeConfig.theme ?? "v2"}>
```

Comment:
And at some point we'll move this up to encapsulate the nav, yah?

Reply:
We'd probably never want the nav to actually change it's theme since that would be really confusing. I can see us changing it to v3 sometime soon but not changing it based on route.